scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -162,20 +165,22 @@ workflow CNVSomaticPairWorkflow {
     }
 
     Int collect_counts_normal_disk = normal_bam_size + ceil(size(PreprocessIntervals.preprocessed_intervals, "GB")) + disk_pad
-    call CNVTasks.CollectCounts as CollectCountsNormal {
-        input:
-            intervals = PreprocessIntervals.preprocessed_intervals,
-            bam = normal_bam,
-            bam_idx = normal_bam_idx,
-            ref_fasta = ref_fasta,
-            ref_fasta_fai = ref_fasta_fai,
-            ref_fasta_dict = ref_fasta_dict,
-            format = format,
-            gatk4_jar_override = gatk4_jar_override,
-            gatk_docker = gatk_docker,
-            mem_gb = mem_gb_for_collect_counts,
-            disk_space_gb = collect_counts_normal_disk,
-            preemptible_attempts = preemptible_attempts
+    if (defined(normal_bam)) {
```

Comment:
Anything stopping you from putting all of the tasks for the normal in a single `if` block?  (If so, no biggie.)

Reply:
@samuelklee I just wanted to preserve the order.  My logic was simply so that we keep the like tasks together.  If you would like me to change it, no problem.  Just tell me.

Reply:
I think it's fine to put them all in the same block.  At some point, we can extract a single-sample subworkflow that takes allelic counts from a matched normal as an optional input to the ModelSegments task (since that is the only difference between running the normal and the tumor).  This pair workflow will just be two calls to that single-sample workflow.  However, this can definitely wait until Cromwell handles subworkflows more elegantly.